pkg/apis/core/typeconfig/interface.go

```diff
@@ -31,4 +31,5 @@ type Interface interface {
 	GetPlacement() metav1.APIResource
 	GetOverride() *metav1.APIResource
 	GetOverridePath() []string
+	GetCollectStatus() bool
```

Comment:
Can you move this commit ahead of https://github.com/kubernetes-sigs/federation-v2/pull/247/commits/3f94daf2256e73b4bc1014d9441f68dd93767f75

Reply:
its already in the same order as you asked (if you look in git). github may have messed up the order of commits).